package.json

```diff
@@ -33,7 +33,7 @@
   "dependencies": {
     "assert-plus": "0.1.5",
     "async": "0.9.0",
-    "bip39": "git://github.com/blocktrail/bip39.git#sjcl-browser",
+    "bip39": "git://github.com/blocktrail/bip39.git#sjcl-browser-bip39",
```

Comment:
FYI: this branch fixes pbkdf2 at 3.0.4. This is the last branch to support v0.10.